### DISCUSSION TITLE
Freeze at the start of the beatmap

### DISCUSSION THREAD

**Discussion by Irelito** [ppy/osu#23968] [Q&A] [answered]
Hi, so I'm getting random freezes only on the start of the beatmap (during gameplay it's completely fine).

Here's my log folder:
[Logs.zip](https://github.com/ppy/osu/files/11790610/Logs.zip)

Here's a video showing an example:

https://www.youtube.com/watch?v=QJyKA2dBvW0

Some additional details:

- This happens mostly when the audio file starts after the map loads in
- After getting a clean Osu install I get the same exact issue
- For some reason in the new install I still get the freeze when I start a map, but I do not get it/get a much smaller freeze on retrying it (with the original install it's the same when starting/retrying)

Just in case, here are the logs from the new installation as well (it's completely fresh, only has the same map shown in the video):
[Logs.zip](https://github.com/ppy/osu/files/11790719/Logs.zip)

Any help would be appreciated, thank you!

P.S.: version b20230326, Windows 11, using an audio interface as well if that matters. Also, this didn't happen before I repaired Windows a month or two ago.

**Comment by Axilotl17** [comment, marked as answer]
This GitHub repository is for osu!Lazer - osu!Stable issues go [here](https://github.com/ppy/osu-stable-issues)
